packages/web/src/pages/upload-page/fields/SourceFilesModalForm.module.css

```diff
@@ -0,0 +1,41 @@
+.header {
```

Comment:
i think default is 2 spaces, run `npm run stylelint` to find out. might want to update vscode for lint on save here, can help with that :)

Reply:
Ah true. Thanks!

As in we have setup for this and mine is broken or like we can add it?

Reply:
I think your setup is maybe broken and can help you set that up. But yeah to fix,run stylelint commands

Reply:
I think I fixed it. Added the stylelint plugin and it's running on save. Thanks!